sql-plugin/src/main/scala/com/nvidia/spark/rapids/RegexParser.scala

```diff
@@ -467,6 +467,39 @@ class CudfRegexTranspiler(mode: RegexMode) {
     cudfRegex.toRegexString
   }
 
+  private def isRepetition(e: RegexAST): Boolean = {
+    e match {
+      case RegexRepetition(_, _) => true
+      case RegexGroup(_, term) => isRepetition(term)
+      case RegexSequence(parts) if parts.nonEmpty => isRepetition(parts.last)
+      case _ => false
+    }
+  }
+
+  private def isSupportedRepetitionBase(e: RegexAST): Boolean = {
+    e match {
+      case RegexEscaped(ch) if ch != 'd' && ch != 'w' => // example: "\B?"
```

Comment:
Nit: Comments should be consistent
```suggestion
      case RegexEscaped(ch) if ch != 'd' && ch != 'w' =>
        // example: "\B?"
```